### ISSUE TITLE
Selected Elements in "Format For" and "Paper Size" drop down are not having 4.5:1 color contrast ratio.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#279]
### Steps to Reproduce

1. Launch Xamrin Work application. It will open "Create a new workbook" window.
2. Navigate to File-> Page Setup and activate it.
3. Take a screenshot of Page setup having selected "Format For / Paper Size" dropdown element. Check the color contrast ration of selected text in the dropdown.

### Expected Behavior
Color contrast ratio should be minimum 4.5:1

### Actual Behavior
Color contrast ratio is less than 4.5:1
as shown in the attached screen-shots.

### User Impact:
Low vision user will feel difficulty in seeing the text.

### Environment
MAC: macOS High Sierra [10.13.4]
Workbook - 1.4 (master/b3f92f9) 

### Log File

![page setup](https://user-images.githubusercontent.com/22956130/39864471-32b36600-5439-11e8-80a3-f7c02546e0de.png)

**Comment by WiproPSAccTest** [comment]
Please add the following labels to this bug:
A11yMAS
Accessibility
MAS1.4.3
Xamarin-Workbook-MAC
Wipro

**Comment by abock** [comment]
This is a stock macOS sheet and control. We do not influence or control the rendering of this. The system-wide "Graphite" appearance may exhibit 4.5:1 contrast. Regardless, this is not something we can or should address in our code as it comes from macOS itself.

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1834
